main/install/configuration.dist.php

```diff
@@ -1544,6 +1544,7 @@
     ]
 ];
 */
+//$_configuration['ck_editor_allowed_copy_paste_image'] = false;
```

Comment:
3 cambios aquí:
- renombrar a ck_editor_block_image_copy_paste
- añadir una línea de comentario arriba: 
```
// Prevent the use of images copy-paste as base64 in the editor to avoid 
// filling the database with images
```
- mover estas líneas por la línea ~1605, antes de "// KEEP THIS AT THE END"